carsus/io/cmfgen/hdfgen.py

```diff
@@ -0,0 +1,51 @@
+import glob
+import logging
+
+logger = logging.getLogger(__name__)
+
+def hdf_dump(cmfgen_dir, patterns, parser, chunk_size=10, ignore_patterns=[]):
+    """Function to parse and dump the entire CMFGEN database.
+
+    Arguments:
```

Comment:
this needs to be `numpydoc` style.

Reply:
Oh, I did it with the VSCode plugin, I'll try to configure it.